packages/common/src/pipes/date_pipe_config.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+export interface DatePipeConfig {
```

Comment:
Note: this symbol should also become a part of the public API. We'd need to add docs for the symbol itself (with example usages) and the fields.
```suggestion
/**
 * ... docs ...
 *
 * @publicApi
 */
export interface DatePipeConfig {
```

Reply:
Thanks for the comments, I tried my best with the usage notes!

Also I skipped the deprecation commit for now, I should be able to do that sometime this week tho 😄